nginx/nginx.conf

```diff
@@ -1,4 +1,4 @@
-user                    <%= nginxConf.NGINX_USER %> <%= nginxConf.NGINX_GROUP %>;
+user                    clozan01 staff;
```

Comment:
Also, avoid committing `.DS_Store` files. If they aren't being ignored, then add them to the `.gitignore` file